containers/sprm-to-json/test-output-expected/R001_X001_Y001.cells.json

```diff
@@ -0,0 +1 @@
+{"1": {"xy": [5.597482073085154, 87.44011932908812], "poly": [[0.0, 100.5], [4.75, 101.5], [9.5, 100.5], [14.25, 101.5], [15.4142, 98.0858], [14.5, 93.3713], [12.3284, 89.5], [8.9948, 86.4948], [5.5, 83.364], [5.5, 78.1997], [5.5, 73.0355], [5.5, 68.9042], [7.5, 64.5684], [3.6645, 64.8355], [0.7199, 68.7801], [0.0, 73.6461], [0.0, 78.8103], [0.0, 83.9745], [0.0, 89.1387], [0.0, 94.3029], [0.0, 99.4672], [0.0, 100.5]], "genes": {"Actin": 0.0, "CD107a": 15652240.0, "CD11c": 8887983.0}, "factors": {"Mean": "4", "Covariance": "6"}}, "2": {"xy": [9.974957074420859, 117.37263908878519], "poly": [[0.0, 130.5], [5.15, 131.5], [10.3, 130.5], [15.8642, 130.5], [20.1314, 133.3686], [22.5, 128.7855], [26.1719, 125.3281], [23.6069, 122.5], [19.5, 120.164], [17.8028, 115.3028], [16.5, 110.2782], [16.5, 105.8268], [14.3839, 101.5], [9.2339, 100.5], [4.0839, 101.5], [0.0, 101.5661], [0.0, 107.1303], [0.0, 112.6945], [0.0, 118.2587], [0.0, 123.8229], [0.0, 129.3872], [0.0, 130.5]], "genes": {"Actin": 0.0, "CD107a": 22590065.0, "CD11c": 13624668.0}, "factors": {"Mean": "2", "Covariance": "2"}}, "3": {"xy": [7.75703282546032, 634.3265190091385], "poly": [[0.0, 647.5], [4.1768, 646.6768], [8.0858, 648.5], [12.3358, 649.5], [16.1213, 647.3787], [16.409, 642.909], [16.5, 639.0355], [16.9445, 634.5555], [16.8787, 630.3787], [14.9948, 626.4948], [11.4038, 623.9038], [8.5, 621.961], [6.5, 618.1252], [2.4118, 616.9118], [0.0, 618.5819], [0.0, 623.2461], [0.0, 627.9103], [0.0, 632.5745], [0.0, 637.2387], [0.0, 641.9029], [0.0, 646.5672], [0.0, 647.5]], "genes": {"Actin": 0.0, "CD107a": 11608143.0, "CD11c": 8046329.0}, "factors": {"Mean": "6", "Covariance": "2"}}}
```

Comment:
Unless it's deeply nested, and the file size would increase dramatically, I think it makes our lives easier to indent JSON output.